bin_sort/bin_sort.py

```diff
@@ -0,0 +1,84 @@
+"""
+Following is the implementation of bin sort algorithm.
+It will sort the input array with time complexity H(N + K)
```

Comment:
@Monal5031 without `int(..)` it can't properly handle `float`s (beforehand I was told to remove them) .. now I updated `main()` with some `float` values and restored use of `int(..)`